src/kafkaaggregator/testtopic/models.py

```diff
@@ -0,0 +1,19 @@
+import faust
+
+
+class TestTopic(faust.Record, serializer="json"):
+    """Test topic with raw values."""
+
+    __test__ = False
+    time: float
+    value: float
+
+
+class AggTestTopic(faust.Record, serializer="json"):
```

Comment:
So I'm guessing that we'll have to metaprogram the creation of these given the input SAL topics and their schemas? That seems like something we can do.

Reply:
That's right, those will be the destination topics with the aggregated values. We need a Faust record for those and to create them programmatically we'll need the schemas for the SAL topics.  That's my understanding too. One thing about ``faust-avro`` that I'll demostrate next is that it can parse the Faust Records into Avro Schemas, which kind of compensate the work in creating the Faust Records in the first place.  As for the SAL topics, because we already have the schema in the Schema Registry, we don't need a correspondent Faust Record, I hope we can do what they call "manual deserialization" for these source topics (see https://faust.readthedocs.io/en/latest/userguide/models.html#manual-serialization)